opencti-platform/opencti-front/src/private/components/data/tasks/TasksList.jsx

```diff
@@ -258,6 +258,7 @@ const TasksList = ({ data }) => {
                         && (isFilterGroupNotEmpty(filters)
                           ? <TasksFilterValueContainer
                               filters={filters}
+                              entityTypes={['Stix-Core-Object', 'stix-core-relationship', 'Notification']}
```

Comment:
I see heterogeneous casing. We don' need capital letters ?

Reply:
stix-core-relationship is not in capital in this case... yes that's heterogeneous :(

Reply:
we need to refacto all the occurences if we want to homogenize...